.github/workflows/trigger-builds.yaml

```diff
@@ -0,0 +1,47 @@
+name: Trigger workflow for release
```

Comment:
Why does this need to be a separate workflow from `release.yaml`?

It appears that we're passing the same inputs from one workflow to the next.